### ISSUE TITLE
vcf 4.2 support

### ISSUE THREAD

**Issue by isthisthat** [jamescasbon/PyVCF#240]
Hello, do you plan to support vcf 4.2?
I'm getting the following error trying to read a MuTect2 vcf:

    File "/illumina/thirdparty/python/python-2.7.5/lib/python2.7/site-packages/vcf/parser.py", line 159, in read_format
    "One of the FORMAT lines is malformed: %s" % format_string)
    SyntaxError: One of the FORMAT lines is malformed: ##FORMAT=<ID=AD,Number=R,Type=
    Integer,Description="Allelic depths for the ref and alt alleles in the order listed">

Looking at the vcf specs, looks like the `Number=R` option was introduced in vcf 4.2
Thank you.

**Comment by martijnvermaat** [comment]
Yes, we'd like to be complete in 4.2 support, although I'm sure we're not :smiley: 

But `Number=R` should actually [be supported](https://github.com/jamescasbon/PyVCF/blob/master/vcf/parser.py#L68).

**Comment by Redmar-van-den-Berg** [comment]
Quick workaround: change "Number=R" to "Number=."

**Comment by jayaramanp** [comment]
hey guys, 
we love using pyvcf, makes the job so much easier!
But we are having issues with parsing vcf4.2 just like the people above have mentioned(switched to a newer version of GATK, bcftools, samtools). 

I like the quick workaround of changing the header for Number=R to Number=. but it wont work for us when we have clinical pipelines creating 100s of vcf files that need to have their headers changed just for pyvcf to work. 

I see that you guys have a branch for vcf4.2 support and I wanted to know if we could use a stable branch with support for vcf4.2 that does not need us to change our VCF headers. 

Thanks again for this!!

**Comment by Redmar-van-den-Berg** [comment]
You can just use the main branch, like @martijnvermaat posted above. You can also install the latest version from pip, which also handles the Number=R automatically. I guess you're using ubuntu, in which has an outdated version of pyvcf that does not handle Number=R properly.

**Comment by jayaramanp** [comment]
Thank you! 
That worked!

**Comment by jamescasbon** [comment]
Someone should really add vcf4.2 test cases first.  We would then have something to shoot for.
